docs/asciidoc/modules/ROOT/pages/graph-updates/graph-refactoring/merge-nodes.adoc

```diff
@@ -41,6 +41,8 @@ In addition, mergeNodes supports the following config properties:
 |===
 | type | operations
 | mergeRels | true/false: give the possibility to merge relationships with same type and direction.
+| produceSelfRel | true/false: is valid only with `mergeRels:true`. If `true` (default), the self-relationships will be created into new node, if exist, and vice versa.
```

Comment:
I think `produceSelfRel` can be used regardless of `mergeRels` value.
Let's say we have this graph:
```
CREATE (a:A), (b:B)
CREATE (a)-[:T]->(b)
CREATE (a)-[:T]->(b)
```
We call mergeNodes on `A` and `B`, with:

1. `mergeRels: false` and `produceSelfRel: false`, we would get 0 `T` relationship
2. `mergeRels: false` and `produceSelfRel: true`, we would get 2 `T` relationships going from `A` to itself
3. `mergeRels: true` and `produceSelfRel: false`, we would get 0 `T` relationship (?)
4. `mergeRels: true` and `produceSelfRel: true`, we would get 1 `T` relationship

So, it seems `produceSelfRel` has it uses regardless of `mergeRels` value.

Side note: @JMHReif and I discussed point 3 and that particular combination is not easy to reason about. We should clearly document what happens in that case.